website/docs/d/schema_site_anp_epg_domain.html.markdown

```diff
@@ -32,42 +32,42 @@ data "mso_schema_site_anp_epg_domain" "anpEpgDomain" {
 ### domain_dn usage ###
 
 ```hcl
-resource "mso_schema_site_anp_epg_domain" "site_anp_epg_domain" {
+
+resource "mso_schema_site_anp_epg_domain" "example_dn" {
   schema_id = "5c4d9fca270000a101f8094a"
   template_name = "Template1"
   site_id = "5c7c95b25100008f01c1ee3c"
   anp_name = "ANP"
   epg_name = "Web"
   domain_dn = "uni/vmmp-VMware/dom-VMware-ab"
-  
 }
 
 ```
 
 ## Argument Reference ##
 
-* `schema_id` - (Required) SchemaID under which you want to deploy Anp Epg Domain.
-* `site_id` - (Required) SiteID under which you want to deploy Anp Epg Domain.
-* `anp_name` - (Required) Name of Application Network Profiles.
-* `epg_name` - (Required) Name of Endpoint Group to manage.
-* `dn` - (Optional) **Deprecated**. The domain profile name. Use `domain_dn` or `domain_name` in association with `domain_type` and `vmm_domain_type` when it is applicable instead.
-* `domain_dn` - (Optional) The dn of domain. This is required when `domain_name` and `domain_type` are not specified.
-* `domain_name` - (Optional) The domain profile name. This is required when `domain_dn` is not used. This attribute requires `domain_type` and `vmm_domain_type` (when it is applicable) to be set.
-* `domain_type` - (Optional) The type of domain to associate. This is required when `domain_dn` is not used. Choices: [ vmmDomain, l3ExtDomain, l2ExtDomain, physicalDomain, fibreChannelDomain ]
-* `vmm_domain_type` - (Optional) The vmm domain type. This is required when `domain_type` is vmmDomain and `domain_dn` is not used. Choices: [ VMware, Microsoft, Redhat ]
+* `schema_id` - (Required) The schema ID under which the Domain is deployed.
+* `site_id` - (Required) The site ID under which the Domain is deployed.
+* `template_name` - (Required) The template name under which the Domain is deployed.
+* `anp_name` - (Required) The ANP name under which the Domain is deployed.
+* `epg_name` - (Required) The EPG name under which the Domain is deployed.
+* `domain_dn` - (Optional) The DN of the Domain. This is required when `domain_name` and `domain_type` are not specified.
+* `domain_name` - (Optional) The name of the Domain. This is required when `domain_dn` is not used. This attribute requires `domain_type` and `vmm_domain_type` (when it is applicable) to be set.
+* `domain_type` - (Optional)  The type of the Domain. This is required when `domain_dn` is not used. Choices: [ vmmDomain, l3ExtDomain, l2ExtDomain, physicalDomain, fibreChannelDomain ]
+* `vmm_domain_type` - (Optional) The type of the VMM Domain. This is required when `domain_type` is vmmDomain and `domain_dn` is not used. Choices: [ VMware, Microsoft, Redhat ]
```

Comment:
should it be Optional -> Read-only ?

Reply:
no since this is possible input on datasource